### DISCUSSION TITLE
No module named 'pytorch_lightning.lite'

### DISCUSSION THREAD

**Discussion by kumara95** [Lightning-AI/pytorch-lightning#18156] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I have installed the pytroch_lightning version 2.0.5, still getting this error

**Comment by Lokesh-26** [comment]
> I have installed the pytroch_lightning version 2.0.5, still getting this error

Execute the below command:
`pip install pytorch-lightning==1.6.0`
